src/Assignment4/CardComponent/styles-card.css

```diff
@@ -1,11 +1,18 @@
 .CardWrapper {
-  display: inline-block;
   padding: 10px;
   margin-top: 20px;
   margin-left: 20px;
   margin-right: 20px;
   width: 20%;
-  border: 1px black solid;
+  /* border: 1px black solid; */
+  box-shadow: 0 7px 29px #64646f33;
+  cursor: pointer;
```

Comment:
You can keep classes names smallcase. Will be clean to write only components as Pascalcase

Reply:
done